### DISCUSSION TITLE
Libmesh conda compatibility with old moose

### DISCUSSION THREAD

**Discussion by louisBS** [idaholab/moose#17790] [Q&A Getting Started] [answered]
Hi all,

I updated my libmesh ( using conda update -all) and now I cannot compile my old moose using the latest libmesh. It's a bit of a pain for me to rebase my moose by resolving all the conflicts, so I wonder if there's any way for me to downgrade my libmesh version using conda to what it was before ?

Thank you very much,
Louis

**Comment by travismui** [comment, marked as answer]
I believe you can use `conda search -i moose-libmesh` to find the available versions of libmesh for moose and then use `conda install moose-libmesh=<ver>` to install a specific version (i.e. `moose-libmesh=2021.02.15` to roll back one version). This should also roll back any dependencies like `moose-petsc` to their correct versions as well, but there may be other packages that you may also need to roll back separately.
